src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void toDraw();
```

Comment:
```suggestion
    public void draw();
```
we do not use 'to' before verbs in method names.